### PR TITLE
Indent only content

### PR DESCRIPTION
**Pull request by leebyron**
This ensures that if the first line is empty (or if the source is completely empty) that an indentation is not added to the beginning of the file.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -156,13 +157,13 @@ describe( 'MagicString', function () {
 		});
 
 		it( 'should not add characters to empty lines', function () {
-			var s = new MagicString( 'abc\ndef\n\nghi\njkl' );
+			var s = new MagicString( '\n\nabc\ndef\n\nghi\njkl' );
```
Comment (leebyron) [inline test/index.js]: This test fails before the diff

File: test/index.js [hunk 2]
```diff
@@ -95,6 +95,7 @@ describe( 'MagicString', function () {
 			s.indent( '\t' ).prepend( '(function () {\n' ).append( '\n}).call(global);' );
 
 			map = s.generateMap({
+				source: 'input.md',
```
Comment (leebyron) [inline test/index.js]: This is failing in current master, needed to be able to complete the test run

File: test/index.js [hunk 3]
```diff
@@ -558,7 +559,7 @@ describe( 'MagicString.Bundle', function () {
 
 			b.addSource({
 				filename: 'bar.js',
-				content: new MagicString( 'console.log( answer );' )
+				content: new MagicString( '\nconsole.log( answer );' )
```
Comment (leebyron) [inline test/index.js]: added leading new-line to ensure source maps still work correctly.

**Comment by Rich-Harris** [comment]
Good catch, thanks

**Pull request merged**